docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -82,7 +82,7 @@ PUT /idx?master_timeout=1s&timeout=1s
 
```

Comment:
Github won't let me comment above this line, but two lines above this there is:

```
PUT /idx?master_timeout=1s&timeout=1s
{"settings": {"index.routing.allocation.include._name": "non_existent_node"} }
```

That would need to be changed to use `myindex` instead of `idx` also for this PR